src/features/dataset/DatasetNameCommitModal.tsx

```diff
@@ -0,0 +1,72 @@
+// this component contains a the modal for editing a dataset's title and creating a commit for the change
+// we are going to deprecate it in favor of the meta editor, but we may want to bring it back someday
```

Comment:
if this is planning to be used for the `meta.title` & not the dataset name, we should rename this component

Reply:
I removed this, we aren't using it ATM.